PTGLgraphComputation/src/proteingraphs/ComplexGraph.java

```diff
@@ -11,6 +11,7 @@
 import graphdrawing.PageLayout;
 import graphdrawing.DrawTools;
 import graphdrawing.DrawResult;
+import java.awt.*;
 import java.awt.BasicStroke;
 import java.awt.Color;
 import java.awt.Font;
```

Comment:
If it is feasible, just import what you need. From quickly screening the code, I think it is just Rectangle2D. If it is more, then remove the other unnecessary imports.

Reply:
Turns out I did not need it after all, so I removed it again.